Developers/Default.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>

```

Comment:
Please move this file to web-server/templates and rename this file using only lowercase letters.